Godeps/Godeps.json

```diff
@@ -80,11 +80,11 @@
 		},
         {
             "ImportPath": "github.com/apache/incubator-openwhisk-client-go/whisk",
-            "Rev": "HEAD"
+            "Rev": "9d1da78fe9dbed176bcb2f7b6b5416637272d293"
```

Comment:
I think this is the wrong hash to use.
I think the latest hash from the repo is f26793c94fd6883867eaa98ed8249024128cbcc6
https://github.com/apache/incubator-openwhisk-client-go/commit/f26793c94fd6883867eaa98ed8249024128cbcc6

On master branch